test/src/api/global.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+global.WINSTON = require('winston');
+
+const app = require('../../../app/app');
+const databaseUtil = require('../../../app/apis/database-api');
+const sinon = require('sinon');
+const configResource = require('../../../app/resources/config-resource');
+
+/**
+ * Mock config resource so we can run against test database instead of
+ * regular database.
+ * Better doing this here so we don't put test code inside app folder.
+ */
+sinon.stub(configResource, 'getDBConnectionURI', () =>
+  configResource.get('MONGO_TEST_URI') + configResource.get('MONGO_TEST_DB')
+);
+
+/**
+ * Start up the server, clean up the database before running tests.
+ */
+before(function(done) {
+  this.timeout(5000);
```

Comment:
That is a timeout to control the amount of time the server needs to start. This is a global before that starts the server before running tests.
I lowered it down to 1 second and still works on my machine, but may vary depending on the machine.